src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -1829,6 +1729,23 @@ public override bool HandleCall (MethodBody callingMethodBody, MethodReference c
 					break;
 
 				default:
+
+					if (calledMethodDefinition.IsPInvokeImpl) {
+						// Is the PInvoke dangerous?
+						bool comDangerousMethod = IsComInterop (calledMethodDefinition.MethodReturnType, calledMethodDefinition.ReturnType);
+						foreach (ParameterDefinition pd in calledMethodDefinition.Parameters) {
+							TypeReference paramTypeReference = pd.ParameterType;
+							if (paramTypeReference != null) {
```

Comment:
Can it actually happen that the type is null?
I don't think we check this anywhere else....